tests/baselines/reference/checkJsdocTypedefOnlySourceFile.js

```diff
@@ -0,0 +1,18 @@
+//// [0.js]
+// @ts-check
+
+var exports = {};
+
+/**
+ * @typedef {string}
+ */
+exports.SomeName;
```

Comment:
is there a way this typedef can be used? The test should try, if only so we can see the error if it fails.

Reply:
No, it cannot be used, see the additional commit I pushed. The typedef essentially goes ignored due to the added check for `container.symbol` (which is false in files that are not modules).